llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -30115,12 +30126,16 @@ X86TargetLowering::shouldExpandAtomicLoadInIR(LoadInst *LI) const {
   // If this a 64 bit atomic load on a 32-bit target and SSE2 is enabled, we
   // can use movq to do the load. If we have X87 we can load into an 80-bit
   // X87 register and store it to a stack temporary.
```

Comment:
Can you move the comment down to the case it applies to (inside the outer if).  Currently, it binds to both cases which is misleading.

Reply:
Done.